### ISSUE TITLE
Fix event names so they explain what they accomplish

### ISSUE THREAD

**Issue by marcnoon** [UserLevelUp/OpNode#5]
Currently event names are generated so they are very basic.  This is a milestone as one commit per event should be completed.

**Comment by marcnoon** [comment]
Note this is a holder issue for creating one issue for each event handler found.  So you create the issue of a single event handler that needs to be named something other than button12_click to something else like ButtonCreateOpNode or ButtonSaveJSON.

**Comment by Vedant755** [comment]
Can I work on the issue!?

**Comment by marcnoon** [comment]
Yes, I'll add you to this issue.  I think 10 people can work on a single issue.
